zerver/lib/actions.py

```diff
@@ -1096,7 +1096,7 @@ def do_change_realm_subdomain(
     # deactivated. We are creating a deactivated realm using old subdomain and setting
     # it's deactivated redirect to new_subdomain so that we can tell the users that
     # the realm has been moved to a new subdomain.
-    placeholder_realm = do_create_realm(old_subdomain, "placeholder-realm")
+    placeholder_realm = do_create_realm(old_subdomain, f"{realm.name}")
```

Comment:
You don't need an f-string here -- just pass `realm.name`, which is equivalent.

Reply:
@timabbott I had tried realm.name at first, sir. But [`do_create_realm() `](https://github.com/zulip/zulip/blob/main/zerver/lib/actions.py#L4924-L4936) accepts a string data whereas [`realm.name` ](https://github.com/zulip/zulip/blob/main/zerver/models.py#L183-L210)is an `Optional[str]` type. It gives an error during the automated tests by my-py.

Reply:
You can address that with `assert realm.name is not None`.